plugins/router/arris-sb6183

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/python
```

Comment:
I think, you are using python3 syntax here. Thus please clarify this by using `python3`. Additionally it is recommended to allow local overrides of the specific python interpreter (including a virtualenv) by using `#!/usr/bin/env python3` instead.